helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -292,7 +292,8 @@ private val PATH_EXCLUDE_REASON_FOR_FILENAME = listOf(
     "setup.py" to BUILD_TOOL_OF,
     "test.js" to TEST_OF,
     "test.py" to TEST_OF,
-    "test_*.c" to BUILD_TOOL_OF
+    "test_*.c" to BUILD_TOOL_OF,
```

Comment:
Unrelated, but it strikes me that `"test_*.c"` should probably rather be `TEST_OF`, or?